### PR TITLE
Removing problematic Warpable test

### LINKED ISSUE #627: Warpable test failing
### What happened just before the problem occurred

Test for Warpable is breaking.

### PublicLab.org username

@alaxallves

Check https://github.com/publiclab/mapknitter/pull/578/files#r285558211 and https://github.com/publiclab/mapknitter/pull/605#issuecomment-494199024

This test is blocking my work at #605 and consequently a lot of upcoming work.

For bug reports, fill out the above template; for feature requests, you can delete the template.

### PR DESCRIPTION
**Pull request by alaxalves**
Fixes #627 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by alaxalves** [comment]
> # [Codecov](https://codecov.io/gh/publiclab/mapknitter/pull/628?src=pr&el=h1) Report
> > Merging [#628](https://codecov.io/gh/publiclab/mapknitter/pull/628?src=pr&el=desc) into [main](https://codecov.io/gh/publiclab/mapknitter/commit/71b8e317b2830106023326dad5909864fe74af75?src=pr&el=desc) will **not change** coverage.
> > The diff coverage is `n/a`.
> 
> [Impacted file tree graph](https://codecov.io/gh/publiclab/mapknitter/pull/628?src=pr&el=tree)
> 
> ```diff
> @@           Coverage Diff           @@
> ##             main     #628   +/-   ##
> =======================================
>   Coverage   72.19%   72.19%           
> =======================================
>   Files          31       31           
>   Lines        1255     1255           
> =======================================
>   Hits          906      906           
>   Misses        349      349
> ```

@jywarren @sashadev-sky  According to this the test I have removed was not testing anything in our current code base.

File: test/unit/warpable_test.rb [hunk 1]
```diff
@@ -49,16 +49,4 @@ def setup
     assert_not_nil @warp.user_id
     assert_equal @warp.map.user_id, @warp.user_id
   end
-
-  test "try export" do
```
Comment (IgorWilbert) [inline test/unit/warpable_test.rb]: Just a small observation. I guess we should try to make a standard for test naming, and use only either single or double quotes :)

Comment (alaxalves) [reply test/unit/warpable_test.rb]: I'm kinda trying to solve those code pattern issues by enforcing a style guide(rubocop) in #547

Comment (IgorWilbert) [reply test/unit/warpable_test.rb]: Sounds great, thank you!

**Comment by alaxalves** [comment]
@jywarren Could you please take a look at this? It's blocking my other MR at #605

**Comment by cesswairimu** [comment]
@alaxalves check Jeff's comment on https://github.com/publiclab/mapknitter/pull/578#discussion_r287894882.

**Comment by alaxalves** [comment]
> @alaxalves check out Jeff's comment on [#578 (comment)](https://github.com/publiclab/mapknitter/pull/578#discussion_r287894882).

Thnx Cess, I'll close this for now.

**Pull request closed without merging**